shell-switcher.el

```diff
@@ -305,7 +305,8 @@ default) or the other window (if OTHER-WINDOW is t)."
     (when (-in-shell-buffer-p)
       (rswitcher-switch-full -ring))
     (-display-shell-buffer other-window)
-    (-prepare-for-fast-key)))
```

Comment:
I don't understand this as repeating `'` is useful for creating new buffers as well as switching to existing ones. With your change I think we loose the ability to create new buffers.